subworkflows/local/utils_nfcore_seqinspector_pipeline/main.nf

```diff
@@ -103,6 +111,20 @@ workflow PIPELINE_INITIALISATION {
         // }
         .set { ch_samplesheet }
 
+    ch_samplesheet
+        .map {
+            meta, fastqs -> meta.tags
+        }
+        .flatten()
+        .unique()
+        .map { tag_name -> [tag_name.toLowerCase(), tag_name] }
+        .groupTuple()
+        .map {
```

Comment:
That is nifty! Well done!

Reply:
Thanks!

Reply:
Note that from 8779909fe25a35950a999847d0b052b1f188de9c, we assert to only allow lowercase letters in tags when the samplesheet is parsed. We can make it more lenient if we want though.